pkg/cmd/server/kubernetes/master/master_config.go

```diff
@@ -269,9 +270,6 @@ func buildUpstreamGenericConfig(s *kapiserveroptions.ServerRunOptions) (*apiserv
 	if err := s.GenericServerRunOptions.ApplyTo(genericConfig); err != nil {
 		return nil, err
 	}
-	if err := s.Etcd.ApplyTo(genericConfig); err != nil {
```

Comment:
where did this go?

Reply:
Removed, because it's adding second `healthz`. This is applied in `apiserverOptions.Etcd.ApplyWithStorageFactoryTo` in `buildKubeApiserverConfig`